r4/resource-apis/account/api_config.bal

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein is strictly forbidden, unless permitted by WSO2 in accordance with
+// the WSO2 Software License available at: https://wso2.com/licenses/eula/3.2
+// For specific language governing the permissions and limitations under
+// this license, please see the license as well as any agreement you’ve
+// entered into with WSO2 governing the purchase of this software and any
+// associated services.
+//
+//
+// AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Shouldn't say 'DO NOT MODIFY'.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1